hooks/terraform-validate.sh

```diff
@@ -7,9 +7,18 @@ set -e
 # workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
 export PATH=$PATH:/usr/local/bin
 
+# Disable output not usually helpful when running in automation (such as guidance to run plan after init)
+export TF_IS_AUTOMATION=1
+
+# Store and return last failure from validate so this can validate every directory passed before exiting
+VALIDATE_ERROR=
```

Comment:
Shouldn't this default to `0` so the `exit` at the end works correctly if there were no validate errors?

Reply:
It works either way, as without quoting on `exit ${VALIDATE_ERROR}` if nothing returned a non-zero exit code, the result is simply calling `exit` resulting in a zero return code — I'll update this to be explicit though.